### PR TITLE
ADEN-4499 Porvata and Playwire warehouse tracking

### PR DESCRIPTION
**Pull request by fraszczakszymon**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/video/player/playerTracker.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+				'line_item_id': params.lineItemId || '',
+				'creative_id': params.creativeId || '',
+				'vulcan_network': '',
+				'vulcan_advertiser': '',
+				'vulcan_price': ''
+			},
+			vulcanResponse;
+
+		if (params.src) {
+			trackingData.wsi = slotTargeting.getWikiaSlotId(params.slotName, params.src);
```
Comment (dianafa) [inline extensions/wikia/AdEngine/js/video/player/playerTracker.js]: what about: 
```
...
'country': geo.getCountryCode(),
'skin': pageLevelParams.skin,
'wsi': params.src ? slotTargeting.getWikiaSlotId(params.slotName, params.src) : '',
'player': playerName,
...

**Comment by dianafa** [comment]
One nitpick only, besides that LGTM

File: extensions/wikia/AdEngine/js/AdInfoTracker.js [hunk 2]
```diff
@@ -106,23 +106,23 @@ define('ext.wikia.adEngine.adInfoTracker',  [
 	}
 
 	function run() {
+		setAdEnginePvUID();
+		if (mercuryListener) {
+			mercuryListener.onEveryPageChange(function() {
+				win.adEnginePvUID = generateUUID();
```
Comment (bognix) [inline extensions/wikia/AdEngine/js/AdInfoTracker.js:112]: :+1:

**Comment by bognix** [comment]
LGTM

**Pull request merged**